packages/SwingSet/src/kernel/initializeKernel.js

```diff
@@ -89,7 +89,7 @@ export function initializeKernel(config, hostStorage, verbose = false) {
       vatKeeper.setSourceAndOptions({ bundleID }, creationOptions);
       vatKeeper.initializeReapCountdown(creationOptions.reapInterval);
       if (!creationOptions.enableSetup) {
-        kernelKeeper.addToRunQueue(harden({ type: 'startVat', vatID }));
+        kernelKeeper.addToSyscallQueue(harden({ type: 'startVat', vatID }));
```

Comment:
I totally don't understand the concept of "syscall queue", but maybe it's just a really bad name for whatever it actually is.  The run queue is a queue of things that go into vats, i.e., things that will result in deliveries, or things that are done _to_ vats..  Syscalls are things that come out of a vat or done _by_ a vat.  Syscalls are also synchronous, so the idea of a queue involving them makes no sense to me.